base/filesystem.jl

```diff
@@ -38,10 +38,10 @@ export File,
        S_IRGRP, S_IWGRP, S_IXGRP, S_IRWXG,
        S_IROTH, S_IWOTH, S_IXOTH, S_IRWXO
 
-import Base: uvtype, uvhandle, eventloop, fd, position, stat, close,
-            write, read, readavailable, read!, isopen, show,
-            seek, seekend, skip, eof,
-            check_open, _sizeof_uv_fs, uv_error, UVError
+import Base:
+    UVError, _sizeof_uv_fs, check_open, close, cwstring, eof, eventloop, fd,
```

Comment:
since `cwstring` is only defined on windows, should it only be imported on windows? ref #14952